server/lobbyconnection.py

```diff
@@ -938,7 +938,8 @@ async def launch_game(
             # options are. Currently, options for ladder are hardcoded into the
             # client.
             "name": game.name,
-            "init_mode": game.init_mode.value,
+            "game_type": GameType.to_string(self.game_type),
```

Comment:
```suggestion
            "game_type": GameType.to_string(game.game_type),
```
This will fix a lot of the failing tests